core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -474,16 +480,6 @@ describe('Exploration editor page component', function() {
         'Introduction');
     });
 
-    it('should load change list by draft changes successfully',
```

Comment:
Is there a unit test to test the deleted test?

Reply:
other tests are already testing this function + (this test is causing flake, in particular, that's why I deleted this test).

Reply:
@heyimShivam Can you share which test exactly is testing this 
https://github.com/oppia/oppia/blob/40cefaf892508ad4f69411e0369456b16a6caa92/core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts#L359-L363

Reply:
i mean other tests are covering whole file already

Reply:
### this test seems to cover this part too.

![Screenshot from 2022-07-17 05-38-59](https://user-images.githubusercontent.com/77006897/179375707-d465ef47-0df8-491c-8aa1-c19f83ae340c.png)

![Screenshot from 2022-07-17 05-39-22](https://user-images.githubusercontent.com/77006897/179375708-d82ef84a-f2b1-43b3-aedc-a9c5f8923793.png)

### Plus other test are also covering this piece of code:


![Screenshot from 2022-07-17 05-43-57](https://user-images.githubusercontent.com/77006897/179375788-15ab0f81-4ef5-426f-8c8e-e61e5b02f143.png)

Reply:
Edit the following test to test if the draft changes are applied and run the tests at least 10 times to ensure the test pass (Sorry for the trouble)
https://github.com/oppia/oppia/blob/4b17535627f214dfd503c804a9abdff43b9689c1/core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts#L736-L757